view/frontend/templates/instant/facet.phtml

```diff
@@ -1,7 +1,7 @@
 <script type="text/template" id="refinements-lists-item-template">
-    <div class="{{cssClasses.label}} {{#isRefined}}checked{{/isRefined}}">
-        <input class="{{cssClasses.checkbox}}" {{#isRefined}}checked{{/isRefined}} type="checkbox">
+    <label class="{{cssClasses.label}} {{#isRefined}}checked{{/isRefined}}">
+        <input class="{{cssClasses.checkbox}}" {{#isRefined}}checked{{/isRefined}} type="checkbox" value="{{value}}" />
         {{value}}
         <span class="{{cssClasses.count}}">{{count}}</span>
-    </div>
+    </label>
```

Comment:
Just to make sure: this is potentially a breaking change. Is that alright?

Reply:
This is and will be updated in the docs as an advisory :/